STYLE.md

```diff
@@ -11,7 +11,9 @@ We will maintain style-related conventions here; please try to adhere to them wh
   - Pragmas, if any
   - Imports
      - In general, list all imports in the earliest place possible.  For example, the top of file should include all imports required by the file's modules except those that must be imported within the module due to a dependence on module parameters or a requirement to open public.
-     - To the extent possible, group related imports together in order.  List common imports such as `LibraBFT.Prelude`, `LibraBFT.Lemmas` and `LibraBFT.Base.Types` first.
+     - List imports alphabetically where possible.
+     - For modules that depend on earlier imports, list them right after the import they depend on.
```

Comment:
Super pedantic, but an import might depend on multiple previous imports, which makes it impossible to follow this rule.  Also, in some cases it may make less sense to co-locate imports.  I think we can leave that judgment for coders (and code reviewers), rather than trying to dictate this.  I'm happy with the other changes.

Reply:
My specific suggestion is to just remove the second bullet.

Reply:
Errr, I mean the second of the two new ones :smile:.